io/zenoh-transport/tests/transport_whitelist.rs

```diff
@@ -137,3 +137,20 @@ fn transport_whitelist_tcp() {
     // Run
     task::block_on(run(&endpoints));
 }
+
+#[cfg(feature = "transport_shm")]
```

Comment:
I believe there is no need to run this test every time on the CI (it will make increase its duration).
Mark the test as follows:
```
#[test]
#[ignore]
```
It can be then run with `cargo test -- --ignored`.